src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -66,6 +67,11 @@ public class LocalCIDockerService {
     @Value("${artemis.continuous-integration.container-cleanup.cleanup-schedule-minutes:60}")
     private int containerCleanupScheduleMinutes;
 
+    // The image architecture that is supported by the build agent
+    // amd64 is the default value, as this is the architecture of Intel and AMD CPUs, which most systems still use
+    @Value("${artemis.continuous-integration.image-architecture:amd64}")
+    private String imageArchitecture;
```

Comment:
```suggestion
    private final String imageArchitecture = System.getProperty("os.arch");
```

Reply:
Not easily. I've looked it up and apparently the `os.arch` can have a number of different values. Most of them are not the same as the Docker values. The 2 relevant Docker values are `amd64` and `arm64`.
Java can apparently return `x86_64` or `amd64` for amd and also `aarch64` or `ard64` for arm.

Also, considering that Docker supports QEMU, it would be feasible to, in the future, make this an array to allow for a greater support of architectures independent of the systems arch.